bin/conform-templates.php

```diff
@@ -0,0 +1,299 @@
+    }
+}
+
+$docs = [
+    'CODE_OF_CONDUCT.md',
+    'CONTRIBUTING.md',
+    'ISSUE_TEMPLATE.md',
+    'PULL_REQUEST_TEMPLATE.md',
+    'SUPPORT.md',
+];
+
+$replace = [
+    '{org}' => $org,
+    '{repo}' => $repo,
+    '{category}' => strpos($repo, 'zend-expressive') === 0 ? 'expressive' : 'components',
+];
+
+foreach ($docs as $file) {
+    if (file_exists('docs/' . $file)) {
+        unlink('docs/' . $file);
+    }
+
+    $content = file_get_contents(__DIR__ . '/../template/docs/' . $file);
+    $content = strtr($content, $replace);
+
+    file_put_contents('docs/' . $file, $content);
+}
+
+if (file_exists('CONTRIBUTING.md')) {
+    unlink('CONTRIBUTING.md');
+}
+
+if (file_exists('CONDUCT.md')) {
+    unlink('CONDUCT.md');
+}
+
+// Update LICENSE.md
+
+$currentYear = date('Y');
+$year = $currentYear;
+if (file_exists('LICENSE.md')) {
+    $content = file_get_contents('LICENSE.md');
+    if (preg_match('/Copyright \(c\) (\d{4})/', $content, $m)) {
+        $year = $m[1];
+    } else {
+        fwrite(STDERR, 'Cannot match year in current LICENSE.md file. Using current year only.' . PHP_EOL);
+    }
+}
+
+$content = file_get_contents(__DIR__ . '/../template/LICENSE.md');
+$content = str_replace(
+    '{year}',
+    $year < $currentYear ? $year . '-' . $currentYear : $currentYear,
+    $content
+);
+
+file_put_contents('LICENSE.md', $content);
+
+// .coveralls.yml
+file_put_contents('.coveralls.yml', file_get_contents(__DIR__ . '/../template/.coveralls.yml'));
+
+// check if repository has documentation
+$hasDocs = file_exists('mkdocs.yml');
+
+// .gitattributes
+$content = file_get_contents(__DIR__ . '/../template/.gitattributes');
+if (! $hasDocs) {
+    $content = preg_replace('/\/mkdocs\.yml export-ignore\s+/', '', $content);
+}
+if (file_exists('phpbench.json')) {
+    if (file_exists('benchmark') && is_dir('benchmark')) {
+        $content = '/benchmark export-ignore' . PHP_EOL . $content;
+    }
+    if (file_exists('benchmarks') && is_dir('benchmarks')) {
+        $content = '/benchmarks export-ignore' . PHP_EOL . $content;
+    }
+    $content = preg_replace(
+        '/\/phpcs\.xml export-ignore/',
+        '/phpbench.json export-ignore' . PHP_EOL . '/phpcs.xml export-ignore',
+        $content
+    );
+}
+if (file_exists('.docheader')) {
+    if (isset($composer['require-dev']['malukenho/docheader'])) {
+        $content = preg_replace(
+            '/\/\.coveralls.yml export-ignore/',
+            '/.coveralls.yml export-ignore' . PHP_EOL . '/.docheader export-ignore',
+            $content
+        );
+    } else {
+        unlink('.docheader');
+    }
+}
+file_put_contents('.gitattributes', $content);
+
+// .gitignore
+$content = file_get_contents(__DIR__ . '/../template/.gitignore');
+if (! $hasDocs) {
+    $content = preg_replace('/^docs\/html\/\s+/m', '', $content);
+    $content = preg_replace('/^zf-mkdoc-theme\/\s+/m', '', $content);
+    $content = preg_replace('/^zf-mkdoc-theme\.tgz\s+/m', '', $content);
```

Comment:
Do the same as I suggested above: use an alternate regexp delimiter, and use the `s` modifier to capture the trailing EOL with the `\s*` set.

Reply:
Changed to `array_diff`